drivers/battery/sbs_gauge/sbs_gauge.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+#define ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/i2c.h>
+
+/*** Standard Commands ***/
```

Comment:
More of curiosity... what does /*** do?

Reply:
Copied from https://github.com/zephyrproject-rtos/zephyr/blob/33e583b6437e460006e20d04ef822b0954edb586/drivers/sensor/sbs_gauge/sbs_gauge.h#L13

You're right, we don't need the extra `*`.